kotlin/lib/src/main/kotlin/truelayer/signing/HeaderName.kt

```diff
@@ -0,0 +1,19 @@
+package truelayer.signing
+
+internal data class HeaderName(val name: String) {
```

Comment:
Should add a brief bit of internal docs here to clarify that this implementation is for case-insensitive equals/hash